packages/base/any/kernels/5.15-lts/configs/arm64-all/arm64-all.config

```diff
@@ -414,6 +414,7 @@ CONFIG_UNMAP_KERNEL_AT_EL0=y
 CONFIG_RODATA_FULL_DEFAULT_ENABLED=y
 # CONFIG_ARM64_SW_TTBR0_PAN is not set
 CONFIG_ARM64_TAGGED_ADDR_ABI=y
+CONFIG_ARM64_TLB_RANGE=y
```

Comment:
where do these kernel config changes come from, why are they needed? The commit message could use a short explanation why (and there's a typo in there, 5.19 -> 5.10 ;).